packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -778,6 +778,57 @@ let getSingleUseLegacyLogCallback = (logger: ITelemetryLoggerExt, type: string)
 	};
 };
 
+/**
+ * This object holds the parameters necessary for the {@link loadContainerRuntime} function.
+ * @legacy
+ * @alpha
+ */
+export interface LoadContainerRuntimeParams {
+	/**
+	 * Context of the container.
+	 */
+	context: IContainerContext;
+	/**
+	 * Mapping from data store types to their corresponding factories
+	 */
+	registryEntries: NamedFluidDataStoreRegistryEntries;
+	/**
+	 * Pass 'true' if loading from an existing snapshot.
+	 */
+	existing: boolean;
+	/**
+	 * Additional options to be passed to the runtime
+	 */
+	runtimeOptions?: IContainerRuntimeOptions;
+	/**
+	 * runtime services provided with context
```

Comment:
What does it mean for this to be undefined? Can we document that? (e.g. with a `@defaultValue` comment)